src/main.rs

```diff
@@ -1,11 +1,22 @@
 #![warn(clippy::pedantic, clippy::nursery)]
 
+use std::alloc::System;
+
+#[global_allocator]
+static A: System = System;
```

Comment:
Why specify the global allocator? As far as I can tell this is implicit

Reply:
I was following some recommendations that i've found in the https://github.com/johnthagen/min-sized-rust repository. It did work on a project that i was doing, so I use it again. If this is unnecessary, forgive me, I'm really young, and I'm just trying to learn more of rust, so I will search more about rust optmizations methods.